awx/api/serializers.py

```diff
@@ -4681,8 +4681,16 @@ def validate_rrule(self, value):
 class ScheduleSerializer(LaunchConfigurationBaseSerializer, SchedulePreviewSerializer):
     show_capabilities = ['edit', 'delete']
 
-    timezone = serializers.SerializerMethodField()
-    until = serializers.SerializerMethodField()
+    timezone = serializers.SerializerMethodField(
+        help_text=_(
+            'The timezone this schedule runs in. This field is extracted from the RRULE. If the timezone in the RRULE is a link to another timezone this field will show the links name.'
```

Comment:
My suggested text:
'Timezone for the schedule. This field is extracted from the RRULE. If the RRULE's timezone is linked to another timezone, the link will reflect that.